engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/runtime/JobDefinitionSuspensionStateDto.ftl

```diff
@@ -1,60 +1,8 @@
 <#-- Generated From File: camunda-docs-manual/public/reference/rest/job-definition/put-activate-suspend-by-id/index.html -->
 <#macro dto_macro docsUrl="">
 
-<#assign noteMutualExclusive = "Note that only one of `processDefinitionId`, `processDefinitionKey` or a job definition
-                                id (as path parameter) is allowed for a request." >
-<#assign noteProcessDefinitionKey = "Note that this parameter will only be considered 
-                                     in combination with `processDefinitionKey`." >
-
-<@lib.dto>
-    
-    <@lib.property
-        name = "processDefinitionId"
-        type = "string"
-        desc = "The process definition id of the job definitions to activate or suspend.
-        
-                ${noteMutualExclusive}"
-    />
-    
-    <@lib.property
-        name = "processDefinitionKey"
-        type = "string"
-        desc = "The process definition key of the job definitions to activate or suspend.
-        
-                ${noteMutualExclusive}"
-    />
-
-    
-    <@lib.property
-        name = "processDefinitionTenantId"
-        type = "string"
-        desc = "Only activate or suspend job definitions of a process definition which belongs to a
-                tenant with the given id.
-                
-                ${noteProcessDefinitionKey}"
-    />
-
-    
-    <@lib.property
-        name = "processDefinitionWithoutTenantId"
-        type = "string"
-        desc = "Only activate or suspend job definitions of a process definition which belongs to
-                no tenant. Value may only be `true`, as `false` is the default
-                behavior.
-                
-                ${noteProcessDefinitionKey}"
-    />
-
-    
-    <@lib.property
-        name = "suspended"
-        type = "boolean"
-        desc = "A `Boolean` value which indicates whether to activate or suspend the referenced job
-                definitions. When the value is set to `true`, the job
-                definitions will be suspended and when the value is set to `false`,
-                they will be activated."
-    />
 
+<@lib.dto extends="SuspensionStateDto">
     
```

Comment:
I think it would be even better if we mirror this change in the Java code. It will make things cleaner, and we wouldn't have to add a comment to point to the additional `JobDefinitionsSuspensionStateDto.ftl` file.

Otherwise, we should add a comment here to point developers to the new `JobDefinitionsSuspensionStateDto` file, in case we need to update a certain property or add a new one.

Reply:
Added comments to both update DTOs